roles/pulp-devel/tasks/main.yml

```diff
@@ -15,6 +15,7 @@
           - tree
           - wget
           - gnupg
+          - rubygems
```

Comment:
FYI: I tested this by running in a debian:10 container "apt-get install rubygems". It maps to installing the "ruby" package which includes the gem command.